frontend/public/locales/en/apply.json

```diff
@@ -85,6 +85,9 @@
       "paragraph1": "Answering the following questions will not impact your application in the CDCP. Providing a response to the questions below is voluntary. Your input, will help Health Canada evaluate and improve the Canadian Dental Care Plan, including understanding if there are potential access or inequity issues. ",
       "paragraph2": "All responses have an option of \"prefer not to answer\". All responses are kept confidential and are only used for program monitoring, evaluation and reporting purposes.",
       "question1": "1. Where were you born?",
+      "first-nations": "Are you First Nations, Inuk (Inuit), or Métis? First Nations (North American Indian) includes Status and Non-Status Indians. If “Yes,” check all that apply.",
+      "first-nations-blank": "",
```

Comment:
Why does this need to be here? Wouldn't it translate the same for both english and french?

Reply:
we changed the name and the value